### PR TITLE
added validation-GEH-feature

### PR DESCRIPTION
**Pull request by serhii-shyian**

### CODE REVIEW

File: src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
+import org.hibernate.validator.constraints.ISBN;
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.URL;
 
 public record CreateBookRequestDto(
+        @NotEmpty(message = "Title may not be empty")
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java]: Is it for purpose you use NotEmpty but not NotBlank?

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Just have a question

File: src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java (hunk 1 above)
Comment (serhii-shyian) [reply src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java]: I just want to test how it works, as I understand that NotEmpty is better to use with collections and NotBlank with strings, I replaced NotEmpty with NotBlank to use one annotation

**Review by Elena-Bruyako — APPROVED** [review]
well done

**Pull request merged**